cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -0,0 +1,1337 @@
+severity "low"
+category "Cost"
+default_frequency "weekly"
+info(
+  version: "0.1.0",
+  provider: "Azure",
+  service: "Databricks",
+  policy_set: "Databricks",
+  recommendation_type: "Usage Reduction"
+)
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email Address"
+  description "Email addresses of the recipients you wish to notify"
+  default []
+end
+
+parameter "param_subscriptions_list" do
+  label "Subscription Allowed List"
+  type "list"
+  description "Allowed Subscriptions, if empty, all subscriptions will be checked"
+  default []
+end
+
+parameter "param_databricks_workspace_list" do
+  label "Databricks Workspace Allowed List"
+  type "list"
+  description "Allowed Databricks Workspace, if empty, all workspaces will be checked"
+  default []
+end
+
+parameter "param_azure_endpoint" do
+  type "string"
+  label "Azure Endpoint"
+  allowed_values "management.azure.com", "management.chinacloudapi.cn"
+  default "management.azure.com"
+end
+
+parameter "param_stats_check_both" do
+  type "string"
+  category "Statistics"
+  label "Idle/Utilized for both CPU/Memory or either"
+  description "Set whether an instance should be considered idle and/or underutilized only if both CPU and memory are under the thresholds or if either CPU or memory are under. Note: this parameter is only valid when at least one Memory Utilization threshold and one CPU Utilization threshold is NOT set to -1"
+  allowed_values "Both CPU and Memory", "Either CPU or Memory"
+  default "Either CPU or Memory"
+end
+
+parameter "param_stats_threshold" do
+  type "string"
+  category "Statistics"
+  label "Threshold Statistic"
+  description "Statistic to use when determining if an instance is idle/underutilized."
+  allowed_values "Average", "Maximum", "Minimum"
+  default "Average"
+end
+
+parameter "param_stats_interval" do
+  type "string"
+  category "Statistics"
+  label "Statistic Interval"
+  description "Interval to use for the time span.  PT1M=1 minute, PT5M=5 minutes, PT15M=15 minutes, PT30M=30 minutes, PT1H=1 hour, PT6H=6 hours, PT12H=12 hours, P1D=1 day"
+  allowed_values "PT1M","PT5M","PT15M","PT30M","PT1H","PT6H","PT12H","P1D"
+  default "PT5M"
+end
+
+parameter "param_stats_lookback" do
+  type "number"
+  category "Statistics"
+  label "Statistic Lookback Period"
+  description "How many days back to look at utilization metrics for compute resources. This value cannot be set higher than 90 because Azure does not retain metrics for longer than 90 days."
+  default 30
+  min_value 1
+  max_value 30
```

Comment:
This should either be changed to max_value 90 or the description should say 30 days, depending on which of the two is accurate. I believe it's the former.